include/session/config/groups/members.hpp

```diff
@@ -79,14 +81,11 @@ struct member {
     ///
     /// Member variable
     ///
-    /// Flag that is set to indicate to the group that this member is an admin.
+    /// Flag that is set to indicate to the group that this member has been promoted to an admin.
```

Comment:
This wording change seems slightly off, since the group creator never gets promoted to admin, and so this seems to be implying that it should be *false* for the creator because "has been promoted to" will be false.

Reply:
Perhaps "is an admin or has an in-progress promotion to admin"

Reply:
Ah yea, good catch - will update